packages/pages/src/build/build.ts

```diff
@@ -0,0 +1,26 @@
+import shell from "shelljs";
+import { CommandModule } from "yargs";
+import { ProjectFilepaths } from "../common/src/project/structure.js";
+
+type BuildArgs = Pick<ProjectFilepaths, "scope">;
+
+const handler = async ({ scope }: BuildArgs) => {
+  // Pass CLI arguments as env variables to use in vite-plugin
+  if (scope) {
+    process.env.YEXT_PAGES_SCOPE = scope;
```

Comment:
Directly setting a process.env wouldn't have side effects would it? Like it wouldn't bleed into other processes or the terminal? If possible I think I prefer setting them in shell.exec's second parameter

Reply:
directly setting it doesn't get into terminal or other processes, only that specific process and its subprocesses. Sure, I can set it in shell.exec second param.